packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -195,6 +187,9 @@ export const makeZCFZygote = async (
         'string',
         X`invitations must have a description string: ${description}`,
       );
+
+      offerHandler || assert(offerHandler, `offerHandler must be provided`);
```

Comment:
Good change. I have no idea why it was optional.

Reply:
```suggestion
      offerHandler || Fail`offerHandler must be provided`;
```
(where `Fail` is destructured from `assert`)

Reply:
done